src/stores/rootStore.ts

```diff
@@ -21,10 +21,6 @@ export const useRootStore = defineStore('root', {
     businessPhone: null,
     businessPhoneExtension: null,
     corpTypeCd: null,
-    ARFilingYear: null,
-    arMaxDate: null,
-    arMinDate: null,
-    nextARDate: null,
```

Comment:
These are no longer stored here by Todo List. Instead, the AR Filing Year is passed as a route query parameter and the others are taken from the business record (store).